index.js

```diff
@@ -0,0 +1,65 @@
+
+let playerObj = "http://localhost:3000/players/1"
+
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
`DOMContentLoaded` isn't necessary here, since the `<script>` tag has a defer. If it didn't have `defer`, your other querySelector code wouldn't work outside of the DOMContentLoaded function.